symphony/bdk/core/datafeed/real_time_event_listener.py

```diff
@@ -0,0 +1,83 @@
+from abc import ABC, abstractmethod
+
+from symphony.bdk.gen.agent_model.v4_event import V4Event
+from symphony.bdk.gen.agent_model.v4_message_sent import V4MessageSent
+from symphony.bdk.gen.agent_model.v4_initiator import V4Initiator
+from symphony.bdk.gen.agent_model.v4_shared_post import V4SharedPost
+from symphony.bdk.gen.agent_model.v4_instant_message_created import V4InstantMessageCreated
+from symphony.bdk.gen.agent_model.v4_room_created import V4RoomCreated
+from symphony.bdk.gen.agent_model.v4_room_updated import V4RoomUpdated
+from symphony.bdk.gen.agent_model.v4_room_deactivated import V4RoomDeactivated
+from symphony.bdk.gen.agent_model.v4_room_reactivated import V4RoomReactivated
+from symphony.bdk.gen.agent_model.v4_user_requested_to_join_room import V4UserRequestedToJoinRoom
+from symphony.bdk.gen.agent_model.v4_user_joined_room import V4UserJoinedRoom
+from symphony.bdk.gen.agent_model.v4_user_left_room import V4UserLeftRoom
+from symphony.bdk.gen.agent_model.v4_room_member_promoted_to_owner import V4RoomMemberPromotedToOwner
+from symphony.bdk.gen.agent_model.v4_room_member_demoted_from_owner import V4RoomMemberDemotedFromOwner
+from symphony.bdk.gen.agent_model.v4_connection_requested import V4ConnectionRequested
+from symphony.bdk.gen.agent_model.v4_connection_accepted import V4ConnectionAccepted
+from symphony.bdk.gen.agent_model.v4_message_suppressed import V4MessageSuppressed
+from symphony.bdk.gen.agent_model.v4_symphony_elements_action import V4SymphonyElementsAction
+
+
+class RealTimeEventListener:
```

Comment:
I believe this should be an ABC ?

Reply:
The user might want to implement some listeners and not all of them right? 
This is why I didn't use the @abstractmethod decorator to force the implementation of the methods.
Why make it an ABC then ?

Reply:
👌 I get it